codeU3.py

```diff
@@ -0,0 +1,222 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Sat Jun 10 09:17:26 2017
+Exercise 3, version 1
+@author: Luiza
+"""
+
+from collections import deque
+import unittest
+import numpy as np
+
+def getPrefixes(word):
+
+    '''Returns the list of all prefixes of given word
+    including the word itself
+    '''
+
+    return set([word[0:n] for n in range(1, len(word)+1)])
+

```

Comment:
I really like your solution about getting prefixes! However, perhaps it can be a class function within the class Dictionary instead?